conda-envs/environment-test-py37.yml

```diff
@@ -0,0 +1,31 @@
+name: pymc3-test-py37
+channels:
+- conda-forge
+- defaults
+dependencies:
+- aesara>=2.1.0
+- arviz>=0.11.2
+- cachetools>=4.2.1
+- cloudpickle
+- fastprogress>=0.2.0
+- h5py>=2.7
+- ipython>=7.16
+- libblas=*=*mkl
+- mkl-service
+- myst-nb
+- numpy=1.15
```

Comment:
Do we have a ticket about numpy version compatibility? Pinning to such an old version doesn't feel right.

Below it's a >= pin.